nautobot_chatops/sockets/slack.py

```diff
@@ -0,0 +1,216 @@
+"""Websocket Client to receive inbound notifications from Slack, parse them, and enqueue worker actions."""
+
+import asyncio
+import json
+import shlex
+
+from django.conf import settings
+from slack_sdk.socket_mode.aiohttp import SocketModeClient
+from slack_sdk.socket_mode.response import SocketModeResponse
+from slack_sdk.socket_mode.request import SocketModeRequest
+from slack_sdk.web.async_client import AsyncWebClient
+
+from nautobot_chatops.workers import get_commands_registry, commands_help, parse_command_string
+from nautobot_chatops.dispatchers.slack import SlackSocketDispatcher
+from nautobot_chatops.utils import socket_check_and_enqueue_command
+
+
+SLASH_PREFIX = settings.PLUGINS_CONFIG["nautobot_chatops"].get("slack_slash_command_prefix")
```

Comment:
As a general best practice, I'd move this inside `main()` so that it gets evaluated at runtime rather than at import time, making testing easier with things like `@override_settings`.